New branch and translation for term/service (#391)
cncf/glossary: content/it/service.md

---
title: Servizio
status: Feedback Appreciated 

> category: Concept

Reviewer: ```suggestion
category: Concetto
```

---

> Bisogna notare che nel campo dell’informatica, l’uso della parola “servizio” può avere molteplici significati. In questa sezione, ci concentreremo sulla definizione tradizionale: il servizio come un microservizio. Ci sono delle sfumature se i servizi differiscono dai microservizi, e persone diverse possono avere opinioni diverse a riguardo. Per una definizione ad alto livello, li tratteremo allo stesso modo. Si prega di fare riferimento alla definizione dei [microservizi](/it/microservices/).

Reviewer 2: ```suggestion
Bisogna notare che nel campo dell’informatica, l’uso della parola “servizio” può avere molteplici significati. In questa sezione, ci concentreremo sulla definizione tradizionale: il servizio come un microservizio. Se i servizi differiscono dai microservizi, possono registrarsi delle sfumature di significato e persone diverse possono avere opinioni diverse a riguardo. Abbiamo deciso che per proporre una spiegazione ad alto livello, li accumuneremo in un'unica definizione. Si prega di fare riferimento alla definizione dei [microservizi](/it/microservices/).
```

> Bisogna notare che nel campo dell’informatica, l’uso della parola “servizio” può avere molteplici significati. In questa sezione, ci concentreremo sulla definizione tradizionale: il servizio come un microservizio. Ci sono delle sfumature se i servizi differiscono dai microservizi, e persone diverse possono avere opinioni diverse a riguardo. Per una definizione ad alto livello, li tratteremo allo stesso modo. Si prega di fare riferimento alla definizione dei [microservizi](/microservices/).

Reviewer: possiamo mettere il link alla pagina italiana, che fa parte del v1
```suggestion
Bisogna notare che nel campo dell’informatica, l’uso della parola “servizio” può avere molteplici significati. In questa sezione, ci concentreremo sulla definizione tradizionale: il servizio come un microservizio. Ci sono delle sfumature se i servizi differiscono dai microservizi, e persone diverse possono avere opinioni diverse a riguardo. Per una definizione ad alto livello, li tratteremo allo stesso modo. Si prega di fare riferimento alla definizione dei [microservizi](/it/microservices/).
```
